samples/Samples.sln

```diff
@@ -0,0 +1,189 @@
+
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Please remove Samples.sln; all projects should be referenced by the sln in the root.